src/controllers/crate_owner_invitation.rs

```diff
@@ -147,15 +150,12 @@ fn prepare_list(
         raw_invitations.pop();
 
         if let Some(last) = raw_invitations.last() {
-            let mut params = IndexMap::new();
-            params.insert(
-                "seek".into(),
-                crate::controllers::helpers::pagination::encode_seek((
-                    last.crate_id,
-                    last.invited_user_id,
-                ))?,
-            );
-            Some(req.query_with_params(params))
+            let seek_key = crate::controllers::helpers::pagination::encode_seek((
+                last.crate_id,
+                last.invited_user_id,
+            ))?;
+
+            Some(seek_key)
```

Comment:
I changed this a bit to receive the `next_page` value inside the `meta` object which is the `seek` value encoded directly instead of using the prebuilt query parameter. This way I just take the value of the seek encoded cursor directly.

Reply:
unfortunately this is a breaking change to our existing API responses and we need to be quite careful with those. ideally we will implement this without any changes to the backend code.

Reply:
@Turbo87 of course! Let me take this back to the previous state and make sure the client side works as expected.